features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/suggestions/SuggestionsPickerView.kt

```diff
@@ -60,7 +60,7 @@ fun SuggestionsPickerView(
                 when (suggestion) {
                     is ResolvedSuggestion.AtRoom -> "@room"
                     is ResolvedSuggestion.Member -> suggestion.roomMember.userId.value
-                    is ResolvedSuggestion.Alias -> suggestion.roomAlias.value
+                    is ResolvedSuggestion.Alias -> suggestion.roomSummary.roomId
```

Comment:
For parity with the `Member` case above, maybe use the `.value`? But we could also remove the `.value`, for the `Member` case. Anyway this is still OK!

Reply:
Right, added `.value` to get a comparable key 👍🏼